### ISSUE TITLE
Validation while training

### ISSUE THREAD

**Issue by alchemi5t** [keithito/tacotron#297]
I was wondering why this pipeline didn't have an evaluation function to get validation loss to check for overfitting. Or is it there and did i miss it?
